packages/scene-composer/src/components/panels/AddComponentMenu.tsx

```diff
@@ -142,8 +174,11 @@ export const AddComponentMenu: React.FC<AddComponentMenuProps> = ({ onSelect })
         type='action-select'
         onClick={({ uuid }) => {
           switch (uuid) {
-            case ObjectTypes.EntityBinding:
-              handleAddEntityBinding();
+            case ObjectTypes.Animations:
+              handleAddAnimations();
+              break;
+            case ObjectTypes.DataBinding:
```

Comment:
What's the reason for changing the existing logic? We should not be modifying any existing logic.